priv/repo/migrations/20191113114340_add_person_id_to_tag.exs

```diff
@@ -3,7 +3,7 @@ defmodule Auth.Repo.Migrations.AddPersonIdToTag do
 
   def change do
     alter table(:tags) do
-      add :person_id, references(:people, on_delete: :nothing)
+      add :person_id, references(:people, on_delete: :delete_all)
```

Comment:
If I understand correctly the on_delete option, this means that when a person is deleted we are also removing all tags linked to the person. Is this the correct logic?

Reply:
This logic is fine in the `Auth` App because only the "Super Admin" has the ability to delete people and any people created on `Auth` will not be creating Tags/Statuses. On the "Main" app this is undesirable because it can leave "holes" in the content when people are working in teams. Think @ghost users here on GitHub. 🙄